netman/core/objects/__init__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Internap.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class ModelObject(object):
```

Comment:
I think that could be only Model, caus its obviously an object...
Would Bean be more representative?  Maybe a bit java-inspired
I'd agree to both, but without the "Object" in the name